microsoft-edge/webview2/webview2-api-reference.md

```diff
@@ -29,6 +29,7 @@ WebView2 is available for the following frameworks (or platforms) and programmin
 
 * [Win32/C++](/microsoft-edge/webview2/reference/win32/index)
 
+* [JavaScript](/microsoft-edge/webview2/reference/javascript/index)
```

Comment:
It would be helpful here to add some explanatory text to make it clear that this isn't just another language we make the webview2 APIs available in.

```suggestion
Additionally, WebView2 exposes a set of JavaScript APIs to the HTML documents running within WebView2 mostly to facilitate communication with the host application:

* [JavaScript](/microsoft-edge/webview2/reference/javascript/index)
```